functional/test_write.yml

```diff
@@ -80,9 +80,30 @@
 
     - name: Write secret dictionary
       hashivault_write:
+        update: True
+        secret: '{{name_dict}}'
+        data: "{{ dict_value }}"
+    - assert: { that: "{{vault_write.changed}} == True" }
+    - assert: { that: "{{vault_write.rc}} == 0" }
+
```

Comment:
Cool, this will make sure dicts are idempotent, but this change was to support lists. Definitely leave this if there wasn't a idempotent test for dicts already.

Reply:
You can copy these tests and just use something like this for data maybe?
```
data:
  - name: item-one
  - name: item-two
  - name: item-three
```

Reply:
Darn, I apparently forgot what I had just done, sorry about that 🤦 
Now retry...